### DISCUSSION TITLE
No Z - Lift after first 2 layers

### DISCUSSION THREAD

**Discussion by EmperorArthur** [suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config#18] [General]
Out of curiosity, I tried [this model](https://www.printables.com/model/488507-z-hop-based-torture-tests) both with the settings in PR #16 and with the following changes:

* Lift Height 0.4mm
* Only Lift Below 0.4mm
* Disable Retract on Layer Change
* Disable Wipe While Retracting

Both printed fine, except for some bed adhesion issues from dust on the first run.  0.2mm Creality PLA @ 205C, 60C bed temperature.  Stringing seemed to be reduced, but it is hard to judge. Both had some spiderweb thin strands, and finish was about the same.

Per the slicer, print time was reduced from 26 min to 23 min.  I did have "Avoid Crossing Perimeters" enabled though, so that may skew the results.

No lifts, and not Wiping means less jitter, and seems to speed up printing with no loss in quality with PLA.  I would love it if someone else could try these changes and see if they also see an improvement.

**Comment by suchmememanyskill** [comment]
I keep lift z enabled as it does make things like tree supports and parts with small distributed bits more likely to succeed. (Disabled for petg as it does increase stringing by a lot). 

It's maybe a good idea to disable that at some point

**Comment by EmperorArthur** [comment]
I thought so as well, but z-lift breaking parts is supposedly what the model I linked is designed to test.

I did add a decent amount of brim for bed adhesion, which probably offset the lack of z-lift knocking the parts off.
